examples/nlp/run.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright (c) maiot GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.core.pipelines.nlp_pipeline import NLPPipeline
+from zenml.core.steps.trainer.nlp_tokenizers.hf_tokenizer import \
+    TokenizerStep
+from zenml.core.steps.split.random_split import RandomSplit
+from zenml.core.datasources.csv_datasource import CSVDatasource
+from zenml.core.repo.repo import Repository
+from zenml.utils.exceptions import AlreadyExistsException
+from examples.nlp.training.trainer import UrduTrainer
+
+# precious handcrafted Urdu Fake News
+base_path = "/Users/nicholasjunge/workspaces/ml/data/urdu_fake_news/" \
+            "urdu_fake_news.csv"
+
+try:
+    nlp_pipeline = NLPPipeline(name="nlp_test")
```

Comment:
You can leave the name empty here and you dont need the try catch

Reply:
Fixed with 681c0dd